lib/ansible/modules/cloud/azure/azure_rm_containerinstance.py

```diff
@@ -46,7 +46,7 @@
             - present
     ip_address:
         description:
-            - The IP address type of the container group (default is 'none')
+            - The IP address type of the container group (default is 'none').
```

Comment:
- The IP address type of the container group.
- Default is C(none) and creating an instance without public IP.